Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -374,4 +374,4 @@
         "CouldNotDetectEncoding": "Could not detect encoding of file '%s'",
         "ErrorWhileParsingParameter": "There was an error while overriding '%s' parameter because of '%s'."
     }
-}
\ No newline at end of file
+}
```

Comment:
The web editor on GitHub added the extra line at the end. Do you need me to remove it?